account/api/views.py

```diff
@@ -1,16 +1,49 @@
-from account.models import PhonePrefix
-from account.api.serializers import PhonePrefixSerializer
-from django.http import Http404
-from rest_framework.views import APIView
+from django.utils.encoding import force_str
+from django.utils.http import urlsafe_base64_decode
+from rest_framework import generics, status
+from rest_framework.decorators import api_view
+from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
-from rest_framework import status
+from rest_framework.views import APIView
+
+from account.api.serializers import (PhonePrefixSerializer,
+                                     RegistrationSerializer)
+from account.models import CustomUser, PhonePrefix
+from account.tokens import account_activation_token
+
+
+class RegisterView(generics.CreateAPIView):
+    queryset = CustomUser.objects.all()
+    permission_classes = (AllowAny,)
+    serializer_class = RegistrationSerializer
 
 
 class PhonePrefixList(APIView):
     """
     List all news, or create a new news.
     """
+
     def get(self, request, format=None):
         phoneprefix = PhonePrefix.objects.all()
         serializer = PhonePrefixSerializer(phoneprefix, many=True)
         return Response(serializer.data)
+
+
+@api_view(["GET"])
```

Comment:
It is not wrong, but it would be better to use class-based views, instead of function based, as it gives more flexibility